src/cart/_structures_from_macros_.h

```diff
@@ -179,17 +179,4499 @@
+	};
+	struct crt_ctl_get_pid_out {
+	int32_t cgp_pid;
+	int32_t cgp_rc;
+	};
+	extern struct crt_req_format CQF_crt_ctl_get_pid;
+struct crt_proto_query_in {
+	d_iov_t pq_ver;
+	int32_t pq_ver_count;
+	uint32_t pq_base_opc;
+	};
+	struct crt_proto_query_out {
+	uint32_t pq_ver;
+	int32_t pq_rc;
+	};
+	extern struct crt_req_format CQF_crt_proto_query;
+struct crt_ctl_fi_attr_set_in {
+	uint32_t fa_fault_id;
+	uint32_t fa_interval;
+	uint64_t fa_max_faults;
+	uint32_t fa_err_code;
+	uint32_t fa_probability_x;
+	uint32_t fa_probability_y;
+	d_string_t fa_argument;
+	};
+	struct crt_ctl_fi_attr_set_out {
+	int32_t fa_ret;
+	};
+	extern struct crt_req_format CQF_crt_ctl_fi_attr_set;
+struct crt_ctl_fi_toggle_in {
+	_Bool op;
+	};
+	struct crt_ctl_fi_toggle_out {
+	int32_t rc;
+	};
+	extern struct crt_req_format CQF_crt_ctl_fi_toggle;
+void crt_req_destroy(struct crt_rpc_priv *rpc_priv);
+static inline _Boolcrt_rpc_cb_customized(struct crt_context *crt_ctx, crt_rpc_t *rpc_pub){
+	return crt_ctx->cc_rpc_cb != ((void *)0);
+}int crt_rpc_priv_alloc(crt_opcode_t opc, struct crt_rpc_priv **priv_allocated, _Bool forward);
+void crt_rpc_priv_free(struct crt_rpc_priv *rpc_priv);
+int crt_rpc_priv_init(struct crt_rpc_priv *rpc_priv, crt_context_t crt_ctx, _Bool srv_flag);
+void crt_rpc_priv_fini(struct crt_rpc_priv *rpc_priv);
+int crt_req_create_internal(crt_context_t crt_ctx, crt_endpoint_t *tgt_ep, crt_opcode_t opc, _Bool forward, crt_rpc_t **req);
+int crt_internal_rpc_register(void);
+int crt_rpc_common_hdlr(struct crt_rpc_priv *rpc_priv);
+int crt_req_send_internal(struct crt_rpc_priv *rpc_priv);
+static inline _Boolcrt_req_timedout(struct crt_rpc_priv *rpc_priv){
+	return (rpc_priv->crp_state == RPC_STATE_REQ_SENT || rpc_priv->crp_state == RPC_STATE_URI_LOOKUP || rpc_priv->crp_state == RPC_STATE_ADDR_LOOKUP || rpc_priv->crp_state == RPC_STATE_TIMEOUT || rpc_priv->crp_state == RPC_STATE_FWD_UNREACH) && !rpc_priv->crp_in_binheap;
+}static inline uint64_tcrt_set_timeout(struct crt_rpc_priv *rpc_priv){
+	uint32_t timeout_sec;
+	uint64_t sec_diff;
+	do {
+	if (!(rpc_priv != ((void *)0))) do {
+	if (__builtin_expect(crt_iv_logfac_cache[DD_FLAG_DLOG_EMERG_crt_iv_logfac - 1], 0)) {
+	if (crt_iv_logfac_cache[DD_FLAG_DLOG_EMERG_crt_iv_logfac - 1] == (int)0x80000000) {
+	do {
+	(crt_iv_logfac_cache[DD_FLAG_DLOG_EMERG_crt_iv_logfac - 1]) = d_log_check((DLOG_EMERG) | crt_iv_logfac);
+	if (crt_iv_logfac_cache[DD_FLAG_DLOG_EMERG_crt_iv_logfac - 1]) d_log(crt_iv_logfac_cache[DD_FLAG_DLOG_EMERG_crt_iv_logfac - 1], "%s:%d %s() " "Assertion failed " "rpc_priv != NULL" "\n", "src/cart/src/cart/crt_rpc.h", 598, __func__);
+	} while (0);
+	break;
+	} d_log(crt_iv_logfac_cache[DD_FLAG_DLOG_EMERG_crt_iv_logfac - 1], "%s:%d %s() " "Assertion failed " "rpc_priv != NULL" "\n", "src/cart/src/cart/crt_rpc.h", 598, __func__);
+	} } while (0);
+	if (d_alt_assert != ((void *)0)) d_alt_assert((int64_t)(rpc_priv != ((void *)0)), "rpc_priv != NULL", "src/cart/src/cart/crt_rpc.h", 598);
```

Comment:
(style)  line over 80 characters
(style)  trailing statements should be on next line